src/js/api/index.ts

```diff
@@ -16,7 +16,7 @@ host = host.replace(/\/$/, '');
 export const getBuildLogURL = (deploymentId: string): string =>
   `${host}/ci/deployments/${deploymentId}/trace`;
 
-const defaultOptions = {
+const defaultOptions: RequestInit = {
   credentials: 'same-origin',
```

Comment:
IIRC, it's set so that the HTTP Auth username/password is sent to the API. We might be able to remove it once we remove HTTP Auth protection.